playbooks/main.yml

```diff
@@ -43,6 +43,11 @@
       tags:
         - octavia-setup
 
+    - include: rpc-octavia-install-amp-image.yml
```

Comment:
Can this be delegated to `utility_all[0]`? I think there were questions raised about installing `python-openstack` client on `infra1` (around `pip` package clashes). To that end, `rpc-mk8s` now does all its OpenStack setup calls in `utility_all[0]`.